tokio/src/io/driver/mod.rs

```diff
@@ -292,18 +280,11 @@ cfg_not_rt! {
 
 cfg_net! {
     cfg_metrics! {
-       impl Handle {
-            // TODO: Remove this when handle contains `Arc<Inner>` so that we can return
-            // &IoDriverMetrics instead of using a closure.
-            //
-            // Related issue: https://github.com/tokio-rs/tokio/issues/4509
-            pub(crate) fn with_io_driver_metrics<F, R>(&self, f: F) -> Option<R>
-            where
-                F: Fn(&IoDriverMetrics) -> R,
-            {
-                self.inner().map(|inner| f(&inner.metrics))
-            }
-       }
+        impl Handle {
+                pub(crate) fn metrics(&self) -> &IoDriverMetrics {
+                    &self.inner.metrics
+                }
+        }
```

Comment:
The formatting is wonky here.

Reply:
It's really a little strange. I've used rustfmt repeatedly.

![image](https://user-images.githubusercontent.com/17427025/168622794-b331cf1c-a38e-47c4-8aa1-c2ccb03fdff7.png)

Reply:
Could you help me point out where it needs to be adjusted? I didn't see it[sad].

Reply:
done